src/injection/container.ts

```diff
@@ -0,0 +1,28 @@
+import { Container } from "inversify";
+import * as Mongo from "mongodb";
+import { IClient, LevelDB, MongoDB } from "../clients";
+import { IClientConfiguraton, MongoDBConfiguration } from "../configurations";
+import { loadApplicationSettings } from "../utils/configLoader";
+
+/**
+ * Composition root
+ */
+
+/**
+ * Dependancy container
```

Comment:
Typo found. Please change to `Dependency`.

Reply:
Lekker boy